src/lib/shell.html

```diff
@@ -50,7 +50,7 @@
                 <button class="bp-btn-plain bp-btn-loading-download bp-is-invisible"></button>
             </div>
         </div>
-        <div class="bp-content">
+        <div class="bp-content" data-testid="bp-content">
```

Comment:
Are we stripping out data-testid like in elements?

Reply:
Yeah we are, so this will get stripped out but html elements created via javascript won't. But when talking to @jstoffan this wasn't an issue because stripping these out is mainly to decrease bloat in the bundle.